### PR TITLE
add RustNL CFP -- needs style review?

### PR DESCRIPTION
**Pull request by ericseppanen**
I don't think this falls under my editorial area (community links), so I'm submitting it as a separate PR. I'm quite unclear about the proper formatting. In particular, the two dates seem unclear-- should one say "closes" or "deadline" while the other says "conference" or something?

@bennyvasquez @nellshamrell feedback requested.

### CODE REVIEW

**Comment by bennyvasquez** [comment]
@ericseppanen that's a good call! What do you think about this to make it slightly more clear?

```
use this format: 
* [**event name**](link to CFP)| CFP closes YYYY-MM-DD | city,state,country | Date of event YYYY-MM-DD
```

File: draft/2024-02-07-this-week-in-rust.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ Are you a new or experienced speaker looking for a place to share something cool
 <!-- CFPs go here, use this format: * [**event name**](link to CFP)| Date CFP closes in YYYY-MM-DD | city,state,country | Date of event in YYYY-MM-DD -->
 <!-- or if none - *No Calls for papers or presentations were submitted this week.* -->
 
+* [RustNL 2024](https://2024.rustnl.org/) 2024-02-19 | Delft, The Netherlands | 2024-05-07
```
Comment (bennyvasquez) [inline draft/2024-02-07-this-week-in-rust.md:77]:
```suggestion
* [RustNL 2024](https://2024.rustnl.org/) CFP closes 2024-02-19 | Delft, The Netherlands | Event date: 2024-05-07
```

File: draft/2024-02-07-this-week-in-rust.md [hunk 2]
```diff
@@ -74,6 +74,8 @@ Are you a new or experienced speaker looking for a place to share something cool
 <!-- CFPs go here, use this format: * [**event name**](link to CFP)| Date CFP closes in YYYY-MM-DD | city,state,country | Date of event in YYYY-MM-DD -->
 <!-- or if none - *No Calls for papers or presentations were submitted this week.* -->
 
+* [RustNL 2024](https://2024.rustnl.org/) CFP closes 2024-02-19 | Delft, The Netherlands | Event date: 2024-05-07
```
Comment (mariannegoldin) [inline draft/2024-02-07-this-week-in-rust.md:77]:
```suggestion
* [RustNL 2024](https://2024.rustnl.org/) CFP closes 2024-02-19 | Delft, The Netherlands | Event date: 2024-05-07 & 2024-05-08
```

**Comment by mariannegoldin** [comment]
Thank you both! Approving with minor date tweak.

**Review by mariannegoldin — APPROVED** [review]
Approved.

**Pull request merged**